state/service.proto

```diff
@@ -95,3 +99,27 @@ message DumpVarsRequest {
 message DumpVarsResponse {
     bytes Variables = 1;
 }
+
+// ChangeStateRequest message allows to change current node state of node.
+// To permit access, used server config options.
+// The request should be signed.
```

Comment:
I suggest leaving here only a comment on the body of the message (a brief description of the fields, the need for a signature).

Reply:
Yeah, I’ll remove last 2 lines

Reply:
What you think about “.. contains a new node state”?

Reply:
Sounds laconic and correct

Reply:
done